gm4_calling_bell/data/gm4_calling_bell/functions/tick.mcfunction

```diff
@@ -0,0 +1,4 @@
+
+execute as @a[scores={gm4_calling_bell_rung=1..}] run function gm4_calling_bell:emerald_check
```

Comment:
Instead of a ticking function, can't you use the `item_used_on_block` advancement trigger?

Reply:
I was thinking a 1t clock would be better than potentially activating an advancement trigger many times, in the event someone decides to constantly right click a bell.

Though with an additional check on their score, that would prevent the trigger from happening since the day clock modifies their score.  Would that + the advancement be a better solution than a 1t clock?

Reply:
Yeah maybe you're right. A score check isn't very expensive anyways so maybe okay to leave this.